src/devices/vdml_motors.c

```diff
@@ -17,6 +17,7 @@
 #include "kapi.h"
 #include "pros/colors.h"
 #include "pros/motors.h"
+#include "pros/apix.h"
```

Comment:
why is this here? is this necessary?

Reply:
Since I was told to move the prototypes of 2 of the functions within motors.h into apix.h, so I thought it made sense to import the header into vdml_motors.c since those 2 functions are now prototyped within apix.h. It appears to compile without, so if you'd rather me remove I can do that.

Reply:
Got it! Srry forgot that was part of it lol